diff --git a/dist.ini b/dist.ini
index fb4d83e..a9c6d97 100644
--- a/dist.ini
+++ b/dist.ini
@@ -8,5 +8,3 @@ copyright_year   = 2010
 [@GopherRepellent]
 [Prereqs / RuntimeRequires]
 Dist::Zilla::Stash::PodWeaver = 1.001000
-[Prereqs / RuntimeRecommends]
-Dist::Zilla::Plugin::PodLinkTests = 1.003
diff --git a/lib/Dist/Zilla/PluginBundle/GopherRepellent.pm b/lib/Dist/Zilla/PluginBundle/GopherRepellent.pm
index f3af62d..d3b6e66 100644
--- a/lib/Dist/Zilla/PluginBundle/GopherRepellent.pm
+++ b/lib/Dist/Zilla/PluginBundle/GopherRepellent.pm
@@ -27,7 +27,6 @@ use Dist::Zilla::Plugin::MinimumVersionTests ();
 use Dist::Zilla::Plugin::NextRelease ();
 use Dist::Zilla::Plugin::PkgVersion ();
 use Dist::Zilla::Plugin::PodCoverageTests ();
-### Dist::Zilla::Plugin::PodLinkTests (); # suggested not required
 use Dist::Zilla::Plugin::PodSpellingTests ();
 use Dist::Zilla::Plugin::PodSyntaxTests ();
 use Dist::Zilla::Plugin::PodWeaver ();
@@ -49,7 +48,6 @@ sub _default_attributes {
 		auto_prereqs   => [Bool => 1],
 		fake_release   => [Bool => $ENV{DZIL_FAKERELEASE}],
 		is_task        => [Bool => 0],
-		pod_link_tests => [Bool => 1],
 		releaser       => [Str  => 'UploadToCPAN'],
 		skip_plugins   => [Str  => ''],
 		skip_prereqs   => [Str  => ''],
@@ -127,19 +125,6 @@ sub configure {
 sub _bundled_plugins {
 	my ($self) = @_;
 
-	# optional... it was difficult to install these
-	my $pod_link_tests = $self->pod_link_tests &&
-		eval 'require Dist::Zilla::Plugin::PodLinkTests';
-
-	if( $pod_link_tests ){
-		$pod_link_tests = ['PodLinkTests'];
-	}
-	else {
-		$pod_link_tests = [];
-		$self->log('PodLinkTests disabled -- unable to load')
-			if $self->pod_link_tests;
-	}
-
 	$self->log_fatal("you must not specify both weaver_config and is_task")
 		if $self->is_task and $self->weaver_config ne $self->_bundle_name;
 
@@ -240,7 +225,6 @@ sub _bundled_plugins {
 			PodSyntaxTests
 			PodCoverageTests
 		),
-			@$pod_link_tests,
 		# Test::Pod::Spelling::CommonMistakes ?
 		qw(
 			PodSpellingTests
@@ -318,7 +302,6 @@ Possible options and their default values:
 	auto_prereqs   = 1  ; enable AutoPrereqs
 	fake_release   = 0  ; if true will use FakeRelease instead of 'releaser'
 	is_task        = 0  ; set to true to use TaskWeaver instead of PodWeaver
-	pod_link_tests = 1  ; use the PodLinkTests plugin if available
 	releaser       = UploadToCPAN
 	skip_plugins   =    ; default empty; a regexp of plugin names to exclude
 	skip_prereqs   =    ; default empty; corresponds to AutoPrereqs:skip
